js/ui/number_box/number_box.mask.js

```diff
@@ -243,6 +243,11 @@ const NumberBoxMask = NumberBoxBase.inherit({
             if(this._parsedValue < 0 || 1 / this._parsedValue === -Infinity) {
                 this._revertSign(e);
                 this._setTextByParsedValue();
```

Comment:
Why is it necessary to call "_setTextByParsedValue" method? Seems to be enough to do revertSign only there and add return there.

Reply:
it sets text in input with parsed value ) pretty sure there are tests for that

Reply:
If remove it and past `return` `_setTextByParsedValue` method will be call with parsed value anyway and will called onValueChanged event without adding that changes

Reply:
plz take a look at failed tests if you do so. They are self-explanatory

Reply:
I have only one failed test 'should be triggered on keyup when valueChangeEvent="keyup"' - and it is worked in real example

Reply:
had i call with Yulia, resolved